openpathsampling/netcdfplus/dictify.py

```diff
@@ -205,7 +211,7 @@ def to_json(self, obj, base_type=''):
         return json.dumps(simplified)
 
     def to_json_object(self, obj):
-        if hasattr(obj, 'base_cls'):
+        if hasattr(obj, 'base_cls') and type(obj) is not type and type(obj) is not abc.ABCMeta:
```

Comment:
The is something that is cause by the use of a metaclass to implements abc. In the classes that use abstract I placed a `__metaclass__ = abc.ABCMeta` which replaces the function `type` by the function `abc.ABCMeta` as the class constructor (not to be confused with the class instance constructor which is the `__init__` method). When building any class that is a potential subclass of this abstract base class the metaclass is also inherited. That means that all subclasses lead to `type(class) == abc.ABCMeta` and not `type(class) == type`. It took me some time to figure that out. 

This part was not needed before since usually we do not store a class, but rather instances and their attributes. The reference to a class was only necessary to provide a store with the possibility to store for which class this store is meant. This is necessary for all object_stores. 

If you have a better idea to check if an obj is a class or an abstract class that would be great. The more pythonic way is probably to use `inspect.isclass()` but that does more or less the same thing and check for more cases and is potentially slower. But I can change that to remore the necessity to import `abc`.